extensions/wikia/PortabilityDashboard/PortabilityDashboardModel.class.php

```diff
@@ -47,7 +47,8 @@ private function getRowList() {
 						'traffic' => $row->traffic,
 						'migrationImpact' => $row->migration_impact,
 						'typelessTemplatesCount' => $row->typeless,
-						'customInfoboxesCount' => $row->custom_infoboxes
+						'customInfoboxesCount' => $row->custom_infoboxes,
+						'lastUpdated' => date('d-m-Y', strtotime($row->updated))
```

Comment:
This is a different behavior than on most special pages - it is a lot more precise, though I wonder how well it meets the Vanguard expectations. Did you contact anyone from this group to check-in?